reference-implementation/to-upstream-wpts/writable-streams/aborting.js

```diff
@@ -203,3 +203,19 @@ test(() => {
     assert_array_equals(ws.events, ['close']);
   });
 }, 'WritableStream should call underlying sink\'s close if no abort is supplied');
+
+promise_test(() => {
+  let thenCalled = false;
+  const ws = new WritableStream({
+    abort() {
+      return {
+        then(resolve) {
```

Comment:
Nit: the proper argument names are `(onFulfilled, onRejected)` not `(resolve, reject)`